orderbook/src/conversions.rs

```diff
@@ -51,6 +52,30 @@ pub fn h256_from_vec(vec: Vec<u8>) -> Result<H256> {
     Ok(H256::from(array))
 }
 
+fn bigint_03_to_04(input: bigdecimal::num_bigint::BigInt) -> BigInt {
+    let (sign, digits) = input.to_u32_digits();
+    BigInt::new(
+        match sign {
+            bigdecimal::num_bigint::Sign::Minus => Sign::Minus,
+            bigdecimal::num_bigint::Sign::NoSign => Sign::NoSign,
+            bigdecimal::num_bigint::Sign::Plus => Sign::Plus,
+        },
+        digits,
+    )
+}
+
+fn bigint_04_to_03(input: BigInt) -> bigdecimal::num_bigint::BigInt {
+    let (sign, digits) = input.to_u32_digits();
+    bigdecimal::num_bigint::BigInt::new(
+        match sign {
+            Sign::Minus => bigdecimal::num_bigint::Sign::Minus,
+            Sign::NoSign => bigdecimal::num_bigint::Sign::NoSign,
+            Sign::Plus => bigdecimal::num_bigint::Sign::Plus,
+        },
+        digits,
+    )
+}
+
```

Comment:
Oh no! Its back again here. Should the unit tests be moved over or did you decide to leave them out because they were trivial and silly?

Reply:
Yes I felt the tests didn't add value here.